.swiftlint.yml

```diff
@@ -86,3 +86,7 @@ custom_rules:
     match_kinds:
       - identifier
     message: "Plain constraints methods are forbidden. Use `pin(...)` alternative"
+  coredata_date_forbidden:
+    included: "Sources/StreamChat/Database/DTOs"
+    regex: "@NSManaged(| \\S*)* var \\S*: (NS)?Date"
```

Comment:
Matches all these variants:
```
    @NSManaged var test1: Date
    @NSManaged private(set) var test2: NSDate
    @NSManaged public private(set) var test23: NSDate
```

Reply:
@polqf Do we need to included paths? I'm concerned that if we change the folder structure of the SDK this stops working. Can we simply discard this, or maybe limit files that end in "DTO.swift" ? 🤔

Reply:
I did it to avoid having an expensive Regex running through the whole code 🤔
I guess it is a trade off, not sure which one I prefer 🙃 thoughts?

Reply:
Let's keep the current one, for now, 👍 I think it is fine 🙂 We can change this later if we see any problems with it